eng/pipelines/windows.yml

```diff
@@ -107,6 +107,8 @@ stages:
           name: Hosted VS2017
 
         submitToHelix: true
+        # Temporary till we reduced workloads on ARM64
+        timeoutInMinutes: 180
```

Comment:
Please include the tracking issue number to undo this in the comment, especially since this says "temporary".

Reply:
done